dd-sdk-android-gradle-plugin/build.gradle.kts

```diff
@@ -63,12 +65,32 @@ publishingConfig("Plugin to upload Proguard/R8 mapping files to Datadog.")
 gradlePlugin {
     plugins {
         register("dd-sdk-android-gradle-plugin") {
-            id = "dd-sdk-android-gradle-plugin" // the alias
+            id = "com.datadoghq.dd-sdk-android-gradle-plugin" // the alias
```

Comment:
with that change is plugin still available using the old syntax `apply plugin: "dd-sdk-android-gradle-plugin"`?

Reply:
Yep, updated the documentation